presto-hive/src/main/java/com/facebook/presto/hive/HiveClientModule.java

```diff
@@ -299,7 +300,7 @@ public StripeMetadataSource createStripeMetadataSource(OrcCacheConfig orcCacheCo
             exporter.export(generatedNameOf(CacheStatsMBean.class, connectorId + "_StripeFooter"), footerCacheStatsMBean);
             exporter.export(generatedNameOf(CacheStatsMBean.class, connectorId + "_StripeStream"), streamCacheStatsMBean);
         }
-        return stripeMetadataSource;
+        return StripeMetadataSourceFactory.of(stripeMetadataSource);
```

Comment:
Just realized the signature of `of` function is with `src` variable name. Could you submit another PR to quickly fix the nit (`source`)?